### PR TITLE
#5757. add new url pattern, key navigation

### PR DESCRIPTION
**Pull request by jstrR**
## Description

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [x] Bugfix

## Issue

**What is the current behavior?**

#5757 
(resolves all cases except n°4, n°5)

**What is the new behavior?**

New url pattern which resolves the router problem . navigation bar rerender on each story
## Breaking change
**Does this PR introduce a breaking change?** (check one with "x", remove the other)
 - [ ] Yes, and I documented them in migration notes
 - [x] No

### CODE REVIEW

**Comment by jstrR** [comment]
resolves all cases except n°4, n°5

File: web/client/components/geostory/navigation/Navigation.jsx [hunk 1]
```diff
@@ -105,6 +106,7 @@ export default ({
                             <ScrollMenu
                                 items={navigableItems}
                                 currentPage={currentPage}
+                                key={geostoryId}
```
Comment (allyoucanmap) [inline web/client/components/geostory/navigation/Navigation.jsx:109]: @jstrR Even if I suggested you to try to use the key to force the re-render based on story id, I think it's not enough.

This solve the point number three visually on the menu items but following our previous discussion, you pointed out that there is still this [updateCurrentPage](https://github.com/geosolutions-it/MapStore2/blob/f1b1326e41bde2ff29c7383aefa4c81fc58df76f/web/client/components/geostory/layouts/sections/enhancers/currentPageSectionManager.jsx#L43) that read the wrong id from the state.

So the issue is related the fact that the currentStory state is not cleared when a different story is selected in the same session. 

We should:

- [ ] remove this `key={geostoryId}` and the connection of `geostoryId` prop in the related plugin
- [ ] clear the currentStory state in the `loadGeostoryEpic`. @offtherailz suggested to add `.startWith(setCurrentStory({}))` at this [position](https://github.com/geosolutions-it/MapStore2/blob/dddf0be5c42051271cac6d54e179eb1969fe85e0/web/client/epics/geostory.js#L360) 

In this way we ensure that the new state is cleared with the setCurrentStory equal to empty object before the new story request.

Please try the suggested changes and let me know if this solve both: the error on the navbar and the ids used in updateCurrentPage.

File: web/client/plugins/GeoStoryNavigation.jsx [hunk 2]
```diff
@@ -52,6 +54,7 @@ const GeoStoryNavigation = ({
             currentPage={currentPage}
             currentPosition={currentPosition}
             totalItems={totalItems}
+            geostoryId={geostoryId}
```
Comment (allyoucanmap) [inline web/client/plugins/GeoStoryNavigation.jsx:57]: This should also be removed based on this https://github.com/geosolutions-it/MapStore2/pull/5767#discussion_r470463141

**Comment by allyoucanmap** [comment]
please backport this fix to 2020.02.xx

**Pull request merged**